Mail/Views/Thread/Message/MessageBodyView.swift

```diff
@@ -32,7 +32,7 @@ struct MessageBodyView: View {
     @StateObject private var model = WebViewModel()
 
     let presentableBody: PresentableBody
-    let isMessagePreprocessed: Bool
+
     var blockRemoteContent: Bool
     @Binding var displayContentBlockedActionView: Bool
```

Comment:
As you edit this view, could you re-order the properties in the right order? Binding, then let/var properties?